sdks/python/setup.cfg

```diff
@@ -26,3 +26,10 @@ verbosity=2
 # fast_coders_test and typecoders_test.
 exclude=fast_coders_test|typecoders_test
 
+[tool:pytest]
+python_files = *_test.py
```

Comment:
Could you add a comment here explaining what is happening here? (Some human readable explanation for all these options.)